packages/vue-dot/src/elements/ExternalLinks/ExternalLinks.vue

```diff
@@ -243,6 +253,25 @@
 
 			return iconMapping[this.computedPosition.x];
 		}
+
+		removeMarginLeftOrRight(): void {
+			const positionClass = this.right ? 'right-0' : 'left-0';
+            const margLeft= this.$refs.menu.calculatedLeft;
+			if(margLeft !== '12px') {
+				this.menuClassLeftOrRight = 'vd-external-links-menu';
+			}else{
+				this.menuClassLeftOrRight = `vd-external-links-menu ${positionClass}`;
+			}
+		}
+
+		updated(): void {
+			if(this.isClicked){
+				setTimeout(() => {
+					this.removeMarginLeftOrRight();
+				},100);
+				this.isClicked= false;
+			}
+		}
```

Comment:
```suggestion
		setMenuPosition(): void {
			setTimeout(() => this.setMenuClass());
		}
```
J'ai vérifié de mon côté, en effet c'est pas trop possible de se passer d'un `setTimeout`, par contre pas besoin de spécifier un délai
Aussi on peut se passer de l'utilisation du hook `updated` en appelant la méthode lors du clic sur le bouton

Reply:
Good, thx!